content/ansible-tower-advanced/5-tower-cluster-jobs/_index.md

```diff
@@ -57,8 +57,8 @@ Note you used the internal hostname above, when using your browser, you have to
 
   - Now get the job details via the API interface:
 
-      - Login to the API with user `admin` and password `MYSECRETPWD`: `https://student<N>-ansible.<LABID>. events.opentlc.com/api/`
-      - Open the URL `https://student<N>-ansible.<LABID>. events.opentlc.com/api/v2/jobs/<ID>/` where `<ID>` is the number of the job you just looked up in the UI.
+      - Login to the API with user `admin` and password `MYSECRETPWD`: `https://{{< param "external_tower" >}}/api/`
```

Comment:
for consistency it might make sense to also use ${mysecretpwd} or something similar, instead of quoting

Reply:
OK, not now?